packages/server/postgres/migrations/1635354572171_addIntegrationProviderAndToken.ts

```diff
@@ -29,6 +29,8 @@ export async function up() {
     END IF;
     CREATE TABLE IF NOT EXISTS "IntegrationProvider" (
       "id" INT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+      "createdAt" TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP NOT NULL,
+      "updatedAt" TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP NOT NULL,
       "type" "IntegrationProviderTypesEnum" NOT NULL,
```

Comment:
Should we call it a `service`? I think it makes more sense. Type feels more like `oauth2` or `webhook`

Reply:
Provider feels like the right name